src/main/java/com/mongodb/jdbc/MongoStatement.java

```diff
@@ -74,8 +74,6 @@ public ResultSet executeQuery(String sql) throws SQLException {
             return resultSet;
         } catch (MongoExecutionTimeoutException e) {
             throw new SQLTimeoutException(e);
-        } catch (Exception e) {
-            throw new SQLException(e);
```

Comment:
Is there a reason we were doing this before? It just seems to obscure the source of the exception

Reply:
It turns any other exception into a SQLException. Since it compiles without this, I'd say it's fine.

Reply:
per slack discussion, we are only going to catch & convert exceptions that we _know_ could arise, not exceptions which are unexpected